### PR TITLE
OCM-9786 | test: automated cases id:73161,74656

### PR DESCRIPTION
**Pull request by jfrazierRH**
These are excluded until the bug related to OCP-73161 is resolved.

### CODE REVIEW

File: tests/e2e/test_rosacli_cluster.go [hunk 1]
```diff
@@ -420,6 +420,61 @@ var _ = Describe("Edit cluster",
...
 				Expect(CD.Private).To(Equal("Yes"))
 			})
+
+		// Excluded until bug on OCP-73161 is resolved
+		It("can verify delete protection on a rosa cluster - [id:73161]",
+			labels.High, labels.Runtime.Day2, labels.Exclude,
+			func() {
+				By("Enable delete protection on the cluster")
+				deleteProtection := constants.DeleteProtectionEnabled
+				_, err := clusterService.EditCluster(clusterID, "--enable-delete-protection=true", "-y")
```
[the remark this answers is not in the document]
Comment (jfrazierRH) [reply tests/e2e/test_rosacli_cluster.go:430]: Added defer

File: tests/e2e/test_rosacli_cluster.go [hunk 2]
```diff
@@ -420,6 +420,61 @@ var _ = Describe("Edit cluster",
...
+				By("Enable delete protection with invalid values")
+				resp, err := clusterService.EditCluster(clusterID,
+					"--enable-delete-protection=aaa",
+					"-y",
+				)
+				Expect(err).To(HaveOccurred())
+				textData := rosaClient.Parser.TextData.Input(resp).Parse().Tip()
+				Expect(textData).Should(ContainSubstring(
+					`Error: invalid argument "aaa" for "--enable-delete-protection" 
+					flag: strconv.ParseBool: parsing "aaa": invalid syntax`))
```
[the remark this answers is not in the document]
Comment (jfrazierRH) [reply tests/e2e/test_rosacli_cluster.go:470]: Removed the library portion of the string.

File: tests/e2e/test_rosacli_cluster.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (radtriste) [reply tests/e2e/test_rosacli_cluster.go:430]: Please get the original value to set it back in the defer.

File: tests/e2e/test_rosacli_cluster.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (radtriste) [reply tests/e2e/test_rosacli_cluster.go:471]: I still can see it

[the remark this answers is not in the document]
Comment (jfrazierRH) [reply tests/e2e/test_rosacli_cluster.go:471]: Did it for one of the commands, but not the other. Updated now.

File: tests/e2e/test_rosacli_cluster.go [hunk 3]
```diff
@@ -425,14 +425,20 @@ var _ = Describe("Edit cluster",
 		It("can verify delete protection on a rosa cluster - [id:73161]",
 			labels.High, labels.Runtime.Day2, labels.Exclude,
 			func() {
+				By("Get default delete protection value")
```
Comment (radtriste) [inline tests/e2e/test_rosacli_cluster.go:428]:
```suggestion
				By("Get original delete protection value")
```

File: tests/e2e/test_rosacli_cluster.go [hunk 4]
```diff
@@ -425,14 +425,20 @@ var _ = Describe("Edit cluster",
...
+				output, err := clusterService.DescribeClusterAndReflect(clusterID)
+				Expect(err).ToNot(HaveOccurred())
+				deleteProtectionDefault := output.EnableDeleteProtection
```
Comment (radtriste) [inline tests/e2e/test_rosacli_cluster.go:431]:
```suggestion
				originalDeleteProtection := output.EnableDeleteProtection
```

File: tests/e2e/test_rosacli_cluster.go [hunk 5]
```diff
@@ -425,14 +425,20 @@ var _ = Describe("Edit cluster",
...
+				defaultState := fmt.Sprintf("--enable-delete-protection=%s", deleteProtectionDefault)
```
Comment (radtriste) [inline tests/e2e/test_rosacli_cluster.go:432]: I don't think you need a variable for that. Just put it into the defer method call

**Review by radtriste — APPROVED** [review]
/lgtm

**Pull request merged**